pkg/collector/reconcile/configmap_test.go

```diff
@@ -103,8 +103,10 @@ receivers:
         metrics_path: /metrics
         scheme: http
         follow_redirects: true
+        enable_http2: true
```

Comment:
@jaronoff97 could you please review this part? I am not sure sure why this has changed. It seems that bump of the prometheus dependency brought this here.